duckduckgo/iOS
# Subscriptions Base Implementation

Task/Issue URL: https://app.asana.com/0/1204099484721401/1205054784245717

## Description 
- Adds an Alpha Debug build configuration
- Implements the base subscription purchase flow.

## How to test
**⚠️  Needs to be tested on a device with an alpha build and internal access**
1. Confirm there's a new setting for privacy view in settings and tap "Learn More"
2. Signup to a plan using a sandbox user account
3. Confirm that signup is complete and you're taken back to the Privacy Pro Confirmation Page
4. Build the Regular app (Not alpha) and confirm there are no subscription-related settings present



Note: 
Settings page will NOT reflect subscribed services yet.  That's coming up in the next PR.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
